### PR TITLE
Rest api - Add doc for tags

### PR DESCRIPTION
**Pull request by hellosnow**
@vwxyzh @superyyrrzz @qinezh @ansyral @chenkennt

### CODE REVIEW

**Comment by vwxyzh** [comment]
👍

**Comment by superyyrrzz** [comment]
:+1:

File: Documentation/tutorial/intro_rest_api_documentation.md [hunk 1]
```diff
@@ -104,6 +117,94 @@ Footer for `microsoft.com/docfx/Contacts/1.6`
...
+- If all APIs are *not* tagged, each API will not be included in any sections.
+- If the API is specified with *one* tag only, it will show inside this one tag section.
+- If the API is specified with *multiple* tags, it will show inside multiple tag sections.
+- If some APIs are specified with tags while some other APIs are not, the untagged APIs will be organized into one auto generated `Other apis` section.
+
+Specific bookmark could be added to tag section using `x-bookmark-id`, which is Swagger schema extensions following [Specification Extensions](http://swagger.io/specification/#vendorExtensions). If no `x-bookmark-id` is specified, `name` of the tag will be the default bookmark.
+
+For example, the following swagger file defines `Basic` and `Advanced` tags.
+1. Sections in the layout:
+   - `set_contacts` API is tagged with `Advanced` only, then it will only show inside `Advanced` tag section.
```
Comment (qinezh) [inline Documentation/tutorial/intro_rest_api_documentation.md]: => show tag section inside `Advanced` ?

**Comment by qinezh** [comment]
:shipit:

File: Documentation/tutorial/intro_rest_api_documentation.md (hunk 1 above)
Comment (hellosnow) [inline Documentation/tutorial/intro_rest_api_documentation.md]: `Advanced` is the name of the tag section...

---
In reply to: [65512141](https://github.com/dotnet/docfx/pull/349#discussion_r65512141) [](ancestors = 65512141)

Comment (qinezh) [inline Documentation/tutorial/intro_rest_api_documentation.md]: :ok_hand:

File: Documentation/tutorial/intro_rest_api_documentation.md [hunk 2]
```diff
@@ -104,6 +117,94 @@ Footer for `microsoft.com/docfx/Contacts/1.6`
...
+   - `get_contacts` API is tagged with both `Basic` and `Advanced`, then it will show inside both of the tag sections.
+   - `delete_contacts` API is not tagged, it will show inside "Other apis" section.
+2. Bookmarks:
+   - Bookmark of `Basic` tag is `BasicBookmark`, which is defined by `x-bookmark-id`.
+   - Bookmark of `Advanced` tag is `Advanced`, which use `name` by default.
+
+```json
```
Comment (chenkennt) [inline Documentation/tutorial/intro_rest_api_documentation.md]: also give a sample html output

**Comment by chenkennt** [comment]
:+1:

**Comment by hellosnow** [comment]
Merge #349

**Pull request closed without merging**